src/views/TimezoneModal.vue

```diff
@@ -55,17 +71,19 @@ import {
   IonTitle,
   IonToolbar,
   modalController,
-  alertController } from "@ionic/vue";
-import { defineComponent } from "vue";
-import { close, save } from "ionicons/icons";
-import { useStore } from "@/store";
-import { UserService } from "@/services/UserService";
-import { hasError } from '@/utils'
-import { DateTime } from 'luxon';
-
-export default defineComponent({
+  alertController
+ } from "@ionic/vue";
+ import { defineComponent } from "vue";
```

Comment:
Remove all the irrelevant changes from the PR those are not related to the issue, like adding extra lines, changing single-quotes to double quotes and vice-versa, changing indentation, adding space after keywords.